src/vs/workbench/parts/search/browser/openAnythingHandler.ts

```diff
@@ -103,7 +115,15 @@ export class OpenAnythingHandler extends QuickOpenHandler {
 		// Check Cache first
 		let cachedResults = this.getResultsFromCache(searchValue, searchWithRange ? searchWithRange.range : null);
 		if (cachedResults) {
-			return TPromise.as(new QuickOpenModel(cachedResults));
+			timerEvent.data = <ITimerEventData>{
```

Comment:
I find quite some similarities to the ITimerEventData further below in the file, can we extract a method and use it in both places to construct the ITimerEventData?